modules/nf-core/falco/main.nf

```diff
@@ -0,0 +1,59 @@
+process FALCO {
+    tag "$meta.id"
+    label 'process_medium'
+
+
+    conda (params.enable_conda ? "bioconda::falco=1.2.1" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/falco:1.2.1--h867801b_3':
+        'quay.io/biocontainers/falco:1.2.1--h867801b_3' }"
+
+    input:
+    tuple val(meta), path(reads)
+
+    output:
+    tuple val(meta), path("*.html"), emit: html
+    tuple val(meta), path("*.txt") , emit: txt
+    path  "versions.yml"           , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    def prefix = task.ext.prefix ?: "${meta.id}"
+    if (meta.single_end) {
+        """
+        [ ! -f  ${prefix}.fastq.gz ] && ln -s $reads ${prefix}.fastq.gz
+        falco $args --threads $task.cpus ${prefix}.fastq.gz
```

Comment:
I don't understand why you need to rename the input files, can the tool not just take them directly like:
```suggestion
        falco $args --threads $task.cpus ${reads}
```

Reply:
Again, I followed what is written in fastQC. We can change both actually

Reply:
I suspect this is to help multiqc to make meaningful names... however again we should ask to people that make the fastQC module

Reply:
Ah, yes, that may well be the reason.